Add changes for GA from Alberto

tomschr opened the pull request:
This PR contains all the changes for GA (text and graphics).

@abravosuse I've added some remarks with `toms 2022-04-27: change xx`. This makes it (hopefully) easier to identify your changes from your document.

TODO for me: crop some images

----

PDF from review 1 with remarks (without change 8)
[SLES-SAP-trento_color_draft_en.pdf](https://github.com/SUSE/doc-slesforsap/files/8573274/SLES-SAP-trento_color_draft_en.pdf)

---

PDF from review 2 with remarks:
[SLES-SAP-trento_remarks_color_draft_en.pdf](https://github.com/SUSE/doc-slesforsap/files/8582719/SLES-SAP-trento_remarks_color_draft_en.pdf)

---

PDF with comments after integrating Albertos comments:
[SLES-SAP-trento_remarks_color_draft_en.pdf](https://github.com/SUSE/doc-slesforsap/files/8591501/SLES-SAP-trento_remarks_color_draft_en.pdf)

abravosuse reviewed (changes requested):
I have been able to include suggestions for most of the changes directly on the code. But for some reason, there are certain lines in the code where I cannot make any comments. These are the changes that I haven't been able to address directly on the code:
1) login section: do we need it? it's already explained in the agent installation section as it is required to get the API key.
2) monitoring dashboard screenshot: please replace it with screenshot from review #1, change 16
3) pacemaker clusters view screenshot: please replace it with screenshot from review #1, change 21
4) pacemaker cluster settings screenshot: please update caption with CLUSTER SETTINGS
5) connection settings tab screenshot: looks huge in the pdf file

abravosuse reviewed (changes requested):
Toms this is another change I cannot do directly on the code.
Line 343: replace with:
<filename>trento-server-installer</filename> and then execute script

abravosuse reviewed (changes requested):
Another change I cannot make on the code, Toms:
Comment out the giving feedback section and this feature will not be available in the GA version.

tomschr commented on xml/article_sap_trento.xml:
```diff
@@ -221,22 +241,25 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
+      <remark>toms 2022-04-27: change 5</remark>
       <itemizedlist>
         <listitem>
-          <para> Ensure that the Trento component running &k8s; has a
+          <para> The Trento component running &k8s; has
             network access to the &sap; landscape. </para>
         </listitem>
```
Ok, makes sense. Remove listitem.

tomschr commented on xml/article_sap_trento.xml:
```diff
@@ -926,73 +1226,93 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
+        <title>HANA Database details</title>
+        <mediaobject>
+          <imageobject>
+            <imagedata fileref="trento-web-hana-database.png"/>
```
Ahh, right. The `width` attribute is missing.

tomschr commented on xml/article_sap_trento.xml:
```diff
@@ -566,6 +772,7 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
       <title>Home page</title>
       <para> The home page allows you to determine at a glance the Health Status of your &sap; environment. </para>
+      <remark>toms 2022-04-27: change 12</remark>
       <figure xml:id="fig-trento-web-home">
```
Added to the commit 2f04bd3

tomschr commented:
> * login section: do we need it? it's already explained in the agent installation section as it is required to get the API key.

Probably not. It was a leftover from a previous PR. I removed the section.


> * monitoring dashboard screenshot: please replace it with screenshot from review #1, change 16

Hmn, I've updated it already.


> * pacemaker clusters view screenshot: please replace it with screenshot from review #1, change 21

Right. Updated.


> * pacemaker cluster settings screenshot: please update caption with CLUSTER SETTINGS

Ok, done.

 
> * connection settings tab screenshot: looks huge in the pdf file

I've corrected it now. :slightly_smiling_face:

abravosuse reviewed (changes requested):
@tomschr the new images/src/png/trento-web-hosts-dashboard-cpu-memory.png screenshot is wrong. It should be:
![image](https://user-images.githubusercontent.com/91731013/165963237-eda0d94f-ca6d-4d7a-b87f-b371d184a8d0.png)

tomschr commented:
> @tomschr the new images/src/png/trento-web-hosts-dashboard-cpu-memory.png screenshot is wrong. It should be:

Thanks, I've updated it now.

abravosuse commented on xml/article_sap_trento.xml:
```diff
@@ -745,95 +979,124 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
           </itemizedlist>
-          <para> Do not forget to save your changes before closing the
-            window. </para>
+          <para> <remark> alberto: from review #2, change 2 </remark> Whenever a change is made and saved in either tab, if any check is selected a button will show up allowing the user to start the check execution right away: <remark> alberto: include last screenshot in review #2, change 2 </remark> </para>
```
Please add the following screenshot after the paragraph:
![image](https://user-images.githubusercontent.com/91731013/165967228-ead32be0-a59a-426f-98ab-83a5af1947a4.png)

tomschr replied:
Thanks. Actually when I went through it, I didn't include it because in comparison to the previous screenshot for me it was a minor change (just the button). That wouldn't qualify it for me to include it. Furthermore, as the target audience for this guide are SAP experts, I think they don't miss this screenshot. We can include some text, if needed.

Additionally, keep in mind, we need to update all screenshots. Whenever the team decides to change the wording or the placement we need to update it.

I didn't miss it, but decided not to include it for the above reasons. If you still think this screenshot has some value, I can include it of course.

abravosuse replied:
Fair enough! In that case, please remove the ":" at the end of the paragraph.

abravosuse commented on xml/article_sap_trento.xml:
```diff
@@ -606,16 +800,45 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
+      <itemizedlist>
+        <listitem>
+          <para>
+            For a system Global Health status to be Passing, all components must be in green status.
```
```suggestion
            For a system Global Health status to be Passing, all components must be in passing (green) status.
```

abravosuse commented on xml/article_sap_trento.xml:
```diff
@@ -606,16 +800,45 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
+        </listitem>
+        <listitem>
+          <para>
+            A system with at least one component in critical status and no component in grey status, will be computed as Critical
```
```suggestion
            A system with at least one component in critical (red) status, will be computed as Critical
```

abravosuse commented on xml/article_sap_trento.xml:
```diff
@@ -606,16 +800,45 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
+            in the Global Health section.
+          </para>
+        </listitem>
+        <listitem>
+          <para>
+            A system with at least one component in grey status, will not compute towards the
+            Global Health status.
+          </para>
+        </listitem>
```
```suggestion
```

abravosuse reviewed (changes requested):
Some corrections to the calculation of the global health status.

tomschr commented:
@abravosuse Thanks for your changes. I've integrated all of them. Let me know when it can be published. :slightly_smiling_face:

abravosuse commented on xml/article_sap_trento.xml:
```diff
@@ -221,22 +241,25 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
-          <para> The &t.server; must be reachable from the &sap;
-            administrator's LAN via HTTP (port TCP/80). </para>
+          <para>
+            &t.server; must be able to reach the &t.agent;s hosts via SSH (port TCP/22).
```
```suggestion
            &t.server; must be able to reach the &t.agent;s hosts at ports TCP/22 (SSH) and TCP/9100 (Prometheus).
```

abravosuse reviewed (changes requested):
During testing in GCP we have learned that the Trento Server must be able to reach the Agent hosts at port TCP/9100 for Prometheus to work. Adding it to the list of requirements.

ajaeger commented on xml/article_sap_trento.xml:
```diff
@@ -40,36 +40,42 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
+      The product continues to be under active development but is available for any &suse;
+      customer with a registered &productname; 15 (SP1 or higher). Issues should be reported
+      through <link xlink:href="https://bugzilla.suse.com"/> under product
+      <productname>SUSE Linux Enterprise Server for SAP Applications 15 (SP1 or higher)</productname>
```
@abravosuse , @tomschr 
We cannot reference bugzilla.suse.com as those products are internal, let's talk about the "usual support way". Toms, it's the default way for contacting support, what do we have in other manuals? Let's use that wording...

tomschr replied:
@ajaeger The "usual support way" can be to link to www.suse.com/support (which we do already in some chapters). Would the following change sufficient?

```
The product continues to be under active development, but is available for any &suse;
customer with a registered &productname; 15 (SP1 or higher).
Contact https://www.suse.com/support for further assistence.
```

ajaeger replied:
Works for me, thanks!

abravosuse reviewed (changes requested):
@toms This change is related to the bullet point about K3s. I cannot make a suggestion directly on the code. Please make the following change:
Replace paragraph from line 185 to 188:
<para> If you do not have a &k8s; and you do not need professional support, &suse; recommends using K3s which can be installed on a single host. </para>
With:
<para> If you do not have a Kubernetes enterprise solution and you would like to test Trento, an easy option could be SUSEs K3s project which provides a single node Kubernetes cluster. Please note K3s is not a recommended solution for a stable Trento production but a viable way to get started and test Trento. </para>
Thanks!

tomschr commented:
@abravosuse 

> This change is related to the bullet point about K3s. I cannot make a suggestion directly on the code. Please make the following change: [...]

Thanks, I made the change in commit [67fac1b](https://github.com/SUSE/doc-slesforsap/pull/127/commits/67fac1b439d204572809e3912003c47610520aed).
